packages/block-editor/src/components/media-replace-flow/index.js

```diff
@@ -238,7 +242,6 @@ const MediaReplaceFlow = ( {
 								showSuggestions={ false }
 								onChange={ ( { url } ) => {
 									onSelectURL( url );
-									editMediaButtonRef.current.focus();
```

Comment:
What's the reason for removing this focus handler?

Reply:
@Mamaduka, it's related to [one of the comments](https://github.com/WordPress/gutenberg/pull/68084#pullrequestreview-2515546731) mentioned above.

> Also as a separate issue, I noticed that the focus manipulation that editMediaButtonRef is being used to do is not relevant/appropriate anymore in these recent implementations.